tools/python/maps_generator/README.md

```diff
@@ -8,292 +8,96 @@ maps built by a generator_tool newer than the app.**
 ## What are maps?
 
 Maps are .mwm binary files with special meta-information for rendering, searching, routing, and other use cases.
-Files from [data/borders](https://github.com/organicmaps/organicmaps/tree/master/data/borders) define map boundaries for each individual file. The world is segmented into separate files by these boundaries, with the intent of having managebly small files to download. These files are referred to as *maps* or *countries*. A country is referring to one of these files, not necesarily a geographic country. Also note that there are two special countries called *World* and *WorldCoasts*. These are small simplified maps of the world and coastlines used when other maps have not yet been downloaded.
+Files from [data/borders](https://github.com/organicmaps/organicmaps/tree/master/data/borders) define map boundaries for each individual file. The world is segmented into separate files by these boundaries, with the intent of having managebly small files to download. These files are referred to as *maps* or *countries*. A country is referring to one of these files, not necesarily a geographic country. Also note that there are two special countries called *World* and *WorldCoasts*. These are small simplified maps of the world and coastlines (sea and ocean watercover) used when other maps have not yet been downloaded.
 
 ## Setup
 
-You must have Python version >= 3.6 and complete the following steps:
+You must have Python version >= 3.7 and complete the following steps:
 
-1. Switch to the branch of your app's version (see the note of #maps_generator section).
-   For example, if you use OMaps 9.2.3 you should do:
+1. Switch to the branch of your app's version (see the note of #maps_generator section). E.g.:
 
 ```sh
-git checkout release-92
+git checkout 2023.06.04-13-android
 ```
 
-The app version can be found in the "About" section in the settings menu of OMaps.
+The app version can be found in the "About" section of Organic Maps app.
 
-2. Build and install the generator_tool.
+2. Build the `generator_tool` binary (run from the root of the repo):
 
 ```sh
 ./tools/unix/build_omim.sh -r generator_tool
 ./tools/unix/build_omim.sh -r world_roads_builder_tool
 ```
 
-3. Change directory:
+3. Go to the `maps_generator` directory:
 
 ```sh
-$ cd omim/tools/python/maps_generator
+cd tools/python/maps_generator
 ```
 
-4. Install dependencies:
+4. Install python dependencies:
 
 ```sh
-maps_generator$ pip3 install -r requirements_dev.txt
+pip3 install -r requirements_dev.txt
 ```
 
-5. 1 Make the ini configuration file:
+5. Create a configuration file with defaults:
 
 ```sh
-maps_generator$ cp var/etc/map_generator.ini.default var/etc/map_generator.ini
+cp var/etc/map_generator.ini.default var/etc/map_generator.ini
 ```
 
-6. Edit the ini file:
+6. Read through and edit the configuration file.
```

Comment:
Insert a link to a file in the repo here?

Reply:
In the repo we have a default/template file, which is not supposed to be edited. A user must copy it to the actual local file (.gitignored). And edit it then.

Reply:
The point is to allow anyone to read the file when reading this documentation, even without opening a terminal.

Something like
`6. Read through and edit the [configuration file](https://github.com/organicmaps/organicmaps/blob/master/tools/python/maps_generator/var/etc/map_generator.ini.default).`

Reply:
yeap the link is there already in the previous point 5.